### ISSUE TITLE
Fbot: Add commander node feature as with old fbot

### ISSUE THREAD

**Issue by mik-tf** [threefoldtech/tfgrid-sdk-go#797]
### Feature

The new farmerbot doesn't work like before with a commander node from a never_shutdown node. See below.

### Description

From a farmer on Telegram (see [here](https://t.me/threefoldfarmers/69469)):

> I set it up with a single node set to never_shutdown and this node does indeed stay awake, however there's always a second node in 'up' state which is seemingly picked at random. Previous fbot used the never_shutdown machine as the commanding node while the new bot doesn't appear to do so.

Either the commanding node feature is still available, and in that case there's a bug with this instance of the fbot, or it isn't a feature of this farmerbot, and in that case it would be great to add it.

In short, the feature would set a specific node to stay awake and give the WOL packets to other nodes.

> commander_node: nodeID

and by default the commander node would be set to never_shutdown: true

Might be other ways to do it of course.

**Comment by scottyeager** [comment]
I don't think there's any need for a "special feature". If one or more nodes are set to never shutdown, farmerbot should shut down all the rest of the nodes.

The farmerbot doesn't have any direct influence on which nodes send WoL packets. Any node that's awake at the time of power target change will send them.

**Comment by mik-tf** [comment]
Ok good to know! 

Also Tan Du Gia said it was fixed on their side with the 0.13.23 version
